TaskManagementSystem/src/main/java/com/hillel/task_management_system/TaskManagementSystemApplication.java

```diff
@@ -0,0 +1,133 @@
+package com.hillel.task_management_system;
+
+/**
+ * Завдання: Реалізуйте можливість створювати нові завдання з вказанням назви, опису, терміну виконання та пріоритету.
+ * Користувачі: Додавання ти видалення користувачів
+ * Призначення завдань: Додайте можливість призначати завдання іншим користувачам. Користувачі можуть відстежувати призначені їм завдання та їх статус.
+ * Статус завдань: Реалізуйте різні статуси для завдань (наприклад, "нове", "в роботі", "завершене", "відкладене" тощо) та можливість їх зміни.
+ * Пошук та фільтрація: Додайте можливість шукати та фільтрувати завдання за різними критеріями, такими як статус, пріоритет, термін виконання тощо.
+ * Зберігання завдань та користувачів зробити в колекціях (самостійно обрати необхідні колекціі)
+ * Сервіси з логікою створити як біни та визвати через ApplicationContext.
+ */
+
+import com.hillel.task_management_system.enums.Priority;
+import com.hillel.task_management_system.enums.Status;
+import com.hillel.task_management_system.model.Task;
+import com.hillel.task_management_system.model.User;
+import com.hillel.task_management_system.service.TaskService;
+import com.hillel.task_management_system.service.UserService;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class TaskManagementSystemApplication {
+
+    public static void main(String[] args) {
+        ConfigurableApplicationContext applicationContext =
```

Comment:
Now you shouldn't communicate with services via context beans. Now the communication is going through the API

Reply:
Of course, but I want to see what it is going on in the console too